pe.c

```diff
@@ -1196,6 +1197,9 @@ handle_image (void *data, unsigned int datasize,
 
 	CopyMem(buffer, data, context.SizeOfHeaders);
 
+	/* Flush the instruction cache for the region holding the image */
+	cache_invalidate(buffer, buffer + context.ImageSize);
```

Comment:
This line and the line above suggest to me that the invalidation always occurs, while that's not what the function does.

Reply:
Thanks for the review @frozencemetery. Along with correcting the comment, what do you think about renaming the function "maybe_cache_invalidate()" or "perhaps "arch_cache_invalidate()"?

Reply:
I've pushed a version w/ the `arch_cache_invalidate()` name, but open to other suggestions.